clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -128,6 +137,17 @@
   )
 )
 
+(define-read-only (is-registered-swap-token (swap-token principal))
+  (is-some (map-get? registered-swap-tokens { swap-token: swap-token }))
```

Comment:
Should we ensure that `registered` is set to `true`?

Reply:
I'm don't think it matters since you can't unregister a pair (there is no `map-delete` anywhere)

Reply:
As a result, I think `(is-some (map-get? registered-swap-tokens { swap-token: swap-token }))` is enough as a check, since if it exists in the map, it is registered and it can't be unregistered